sympy/physics/quantum/grover.py

```diff
@@ -291,7 +291,7 @@ def grover_iteration(qstate, oracle):
         >>> basis_states = superposition_basis(numqubits)
         >>> f = lambda qubits: qubits == IntQubit(2)
         >>> v = OracleGate(numqubits, f)
-        >>> qapply(grover_iteration(basis_states, v))
+        >>> qapply(grover_iteration(basis_states, v)).expand()
         |2>
```

Comment:
Also here is `expand` necessary?

Reply:
You are right. This slipped through when I removed the .expand() calls after adding moderate expansion to qapply. Will be removed in the next commit.